backend/src/impl/default_controllers_impl.py

```diff
@@ -401,15 +401,15 @@ def systems_analyses_post(body: SystemsAnalysesBody):
         system1_info_dict = general_to_dict(system1_info)
         system1_output_info = SysOutputInfo.from_dict(system1_info_dict)
 
-        system1_metric_stats: MetricStats = [
-            MetricStats(stat) for stat in systems[0].metric_stats[0]
+        system1_metric_stats: SimpleMetricStats = [
```

Comment:
Should this be `system1_metric_stats: list[SimpleMetricStats]`?

```suggestion
        system1_metric_stats: list[SimpleMetricStats] = [
```